src/java.desktop/share/classes/java/awt/GraphicsEnvironment.java

```diff
@@ -357,9 +357,9 @@ public void preferLocaleFonts() {
     }
 
     /**
-     * Indicates a preference for proportional over non-proportional (e.g.
-     * dual-spaced CJK fonts) fonts in the mapping of logical fonts to
-     * physical fonts. If the default mapping contains fonts for which
+     * Indicates a preference for proportional over non-proportional (for
```

Comment:
This does not use the newer `@summary` tag to allow for simple backporting to JDK8 which has the same issue.